include/aspect/simulator.h

```diff
@@ -582,6 +582,12 @@ namespace aspect
       std::vector<std_cxx11::shared_ptr<internal::Assembly::Assemblers::AssemblerBase<dim> > > assembler_objects;
 
       /**
+       * Will call create_additional_material_model_outputs() functions from
+       * each object in assembler_objects.
```

Comment:
I know I'm coming late to this PR, but as a potential follow-up, could you rewrite the documentation of this function to say *what it does and why*, rather than (or in additional to) *how*? As a reader, the function's description doesn't tell you much why or when one would want to call this function.